src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -465,6 +473,44 @@ private string GetValue(NodeModel node)
             return data;
         }
 
+        private object GenerateWatchData(MirrorData data)
```

Comment:
No, this method supposed to return various types like string, number or array. In this case, in the Flood, we will be fully aware of what information we get, instead of just getting a set of string.
But maybe I overdid it? Maybe we really should just send a list of strings?
@pboyer, @Benglin what do you think?